### ISSUE TITLE
New server wanted in China

### ISSUE THREAD

**Issue by Qinyihao** [t1m0thyj/WinDynamicDesktop#220]
Hi there
I am a user of WinDynamicDesktop in China.
I experienced a very low download speed.
![image](https://user-images.githubusercontent.com/40647091/86473798-2acaa100-bd74-11ea-9220-33bb3cbd1333.png)
I want you to build a server near or (if possible) in China so that users in China can download faster.
Also, I found that lots of users of this software in China are experiencing the same problem.
Hope you can make the download faster.
Wish you a good day.

**Comment by Qinyihao** [comment]
If anyone who use this app in China also experienced low download speed, you are free to comment.

**Comment by IanGe2000** [comment]
I guess you need a VPN server to reroute the download for yourself. That download, based on my simple analysis to the code (which may be wrong, btw), is from live.com, which is blocked by China and maybe by China alone. So this may not be a problem that affects most of the users of this app. Instead of waiting for the service to change for us, I think, it's better if we change to adapt to the service if we want it, for now. (Chinese-users-cry-out-loud-deep-in-heart-but-still-try-to-simile-on-the-face.jpg)

**Comment by IanGe2000** [comment]
If allowed, I can put up a repository of the theme folder.
or...maybe it's already done by someone else in China?

**Comment by t1m0thyj** [comment]
> I want you to build a server near or (if possible) in China so that users in China can download faster.

Since WDD is a free open-source project, I don't have the resources to build a server. However, you can download the Big Sur themes from [here](https://github.com/t1m0thyj/WDD-mac-themes/releases/tag/big-sur) if the app's built-in download is too slow.

> I guess you need a VPN server to reroute the download for yourself.

Using VPN may be help, I believe other users in China have also found this to be true.

> If allowed, I can put up a repository of the theme folder.
> or...maybe it's already done by someone else in China?

There is a repository [here](https://github.com/t1m0thyj/WDD-mac-themes/releases). In the latest version of the app (4.2.0), the default download source should have changed to use GitHub instead of live.com.

**Comment by Qinyihao** [comment]
> I guess you need a VPN server to reroute the download for yourself. That download, based on my simple analysis to the code (which may be wrong, btw), is from live.com, which is blocked by China and maybe by China alone. So this may not be a problem that affects most of the users of this app. Instead of waiting for the service to change for us, I think, it's better if we change to adapt to the service if we want it, for now. (Chinese-users-cry-out-loud-deep-in-heart-but-still-try-to-simile-on-the-face.jpg)

Yes, I used a VPS to build a ShadowSocksR where it's located in NJ in the US.
But, the speed is still too slow.

**Comment by Qinyihao** [comment]
> > I want you to build a server near or (if possible) in China so that users in China can download faster.
> 
> Since WDD is a free open-source project, I don't have the resources to build a server. However, you can download the Big Sur themes from [here](https://github.com/t1m0thyj/WDD-mac-themes/releases/tag/big-sur) if the app's built-in download is too slow.
> 
> > I guess you need a VPN server to reroute the download for yourself.
> 
> Using VPN may help, I believe other users in China have also found this to be true.
> 
> > If allowed, I can put up a repository of the theme folder.
> > or...maybe it's already done by someone else in China?
> 
> There is a repository [here](https://github.com/t1m0thyj/WDD-mac-themes/releases). In the latest version of the app (4.2.0), the default download source should have changed to use GitHub instead of live.com.

Thanks! CDN from GitHub is much faster.
